build.gradle

```diff
@@ -249,6 +243,9 @@ publishing {
 }
 
 artifactoryPublish {
+  onlyIf {
+    System.env.'TRAVIS_BRANCH' == "master" && !System.env.'TRAVIS_PULL_REQUEST'.toBoolean()
```

Comment:
In that case we should be publishing to bintray. See https://github.com/ReadyTalk/gradle-readytalk-js/blob/master/build.gradle